src/lib/ua_overrides.js

```diff
@@ -164,11 +178,20 @@ class UAOverrides {
       if (platformMatches.includes(override.platform)) {
         override.availableOnPlatform = true;
 
-        // If there is a specific experiment running for
-        // this override, get its state.
+        // Note whether the user is actively in the override's experiment (if any).
+        override.experimentActive = false;
         const experiment = override.config.experiment;
-        override.experimentActive =
-          experiment && (await browser.experiments.isActive(experiment));
+        if (experiment) {
+          const branches = Array.isArray(experiment)
```

Comment:
Can we comment on what's happening here? It took my brain too long to parse it.

Reply:
Yep.